code/world.dm

```diff
@@ -269,6 +269,17 @@ var/last_irc_status = 0
 		'sound/roundend/yeehaw.ogg',
 		'sound/roundend/disappointed.ogg'\
 		)
+	
+	var/titlescreen = TITLESCREEN
+	if(!titlescreen)
+		titlescreen = "title"
+
+	for(var/thing in clients)
+		if(thing)
+			var/client/C = thing
+			C.screen.Cut()
+			C.screen += new /obj/screen/splash
```

Comment:
Can someone tell me if there's a way to put this on top without deleting the hud?